topics/RDMA.md

```diff
@@ -0,0 +1,167 @@
+  'ValkeyRdmaCmd'.
+- To receive a control message by RDMA '**ibv_post_recv**', and the received buffer
+  size should be size of 'ValkeyRdmaCmd'.
+- To transfer stream data by RDMA '**ibv_post_send**' with opcode '**IBV_WR_RDMA_WRITE**'(optional) and
+  '**IBV_WR_RDMA_WRITE_WITH_IMM**'(required), to write data segments into a connection by
+  RDMA [WRITE][WRITE][WRITE]...[WRITE WITH IMM], the length of total buffer is described by
+  immediate data(unsigned int 32).
+
+
+### Maximum WQE(s) of RDMA
+Currently no specific restriction is defined in this protocol. Recommended WQEs is 1024.
+Flow control for WQE MAY be defined/implemented in the future.
+
+
+### The workflow of this protocol
+```
+                                                                    valkey-server
+                                                                    listen RDMA port
+   valkey-client
+                -------------------RDMA connect-------------------->
+                                                                    accept connection
+                <--------------- Establish RDMA --------------------
+
+                --------Get server feature [@IBV_WR_SEND] --------->
+
+                --------Set client feature [@IBV_WR_SEND] --------->
+                                                                    setup RX buffer
+                <---- Register transfer memory [@IBV_WR_SEND] ------
+[@ibv_post_recv]
+setup TX buffer
+                ----- Register transfer memory [@IBV_WR_SEND] ----->
+                                                                    [@ibv_post_recv]
+                                                                    setup TX buffer
+                -- Valkey commands [@IBV_WR_RDMA_WRITE_WITH_IMM] -->
+                <- Valkey response [@IBV_WR_RDMA_WRITE_WITH_IMM] ---
+                                  .......
+                -- Valkey commands [@IBV_WR_RDMA_WRITE_WITH_IMM] -->
+                <- Valkey response [@IBV_WR_RDMA_WRITE_WITH_IMM] ---
+                                  .......
+
+
+RX is full
+                ----- Register transfer memory [@IBV_WR_SEND] ----->
+                                                                    [@ibv_post_recv]
+                                                                    setup TX buffer
+                <- Valkey response [@IBV_WR_RDMA_WRITE_WITH_IMM] ---
+                                  .......
+
+                                                                    RX is full
+                <---- Register transfer memory [@IBV_WR_SEND] ------
+[@ibv_post_recv]
+setup TX buffer
+                -- Valkey commands [@IBV_WR_RDMA_WRITE_WITH_IMM] -->
+                <- Valkey response [@IBV_WR_RDMA_WRITE_WITH_IMM] ---
+                                  .......
+
+                -------------------RDMA disconnect----------------->
+                <------------------RDMA disconnect------------------
+```
+
+
+## Getting Started
+---------------
```

Comment:
For a level 2 heading, use one of `##` or `--------`, not both.

```suggestion
## Getting Started

```

Btw, isn't it better to but "Getting started" first and "Protocol" below?